google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -438,6 +439,10 @@ public static InstantiatingGrpcChannelProvider.Builder defaultGrpcTransportProvi
       grpcTransportProviderBuilder
           .setAttemptDirectPathXds()
           .setAttemptDirectPath(true)
+          // Try to fetch a hard-bound access token for direct access if the runtime
+          // environment supports it.
+          .setAllowHardBoundTokenTypes(
```

Comment:
You can set it as default. Not a hard requirement, but could we add a env var to disable this incase we have any problem? Better to use env var rather than rollback client to prev version.

Reply:
This is a great point. Done. Thanks for the tip!